src/main/java/mate/academy/springboot/datajpa/repository/specification/ProductSpecificationManager.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.springboot.datajpa.repository.specification;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import mate.academy.springboot.datajpa.model.Product;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ProductSpecificationManager implements SpecificationManager<Product> {
```

Comment:
If I remember correctly its from Bohdan's video?
I think its used when we get multiply lists in request params and have more complex filtration logic, but in our case we search only by category name so all this logic is not really needed
You can delete all specification package :)